### ISSUE TITLE
Installing AirSonos

### ISSUE THREAD

**Issue by Acili** [npm/npm#10904]
I´m running a Mac OS X El Capitan 10.11.2 and I´m trying to install AirSonos so I can be able to play/stream music directly from youtube through my Sonos soundsystem.

I´m trying in terminal to follow the instructions from here:
https://github.com/stephen/airsonos/blob/master/INSTALL.md

but when I do:

`npm install -g airsonos`

I get the following error-message:

```
npm WARN addRemoteGit git://github.com/stephen/node-sonos.git#stephen-1.0.0 resetting remote /var/root/.npm/_git-remotes/git-github-com-stephen-node-sonos-git-701294ef because of error: xcrun: error: invalid active developer path (/Library/Developer/CommandLineTools), missing xcrun at: /Library/Developer/CommandLineTools/usr/bin/xcrun
npm ERR! git clone --template=/var/root/.npm/_git-remotes/_templates --mirror git://github.com/stephen/node-sonos.git /var/root/.npm/_git-remotes/git-github-com-stephen-node-sonos-git-701294ef: xcrun: error: invalid active developer path (/Library/Developer/CommandLineTools), missing xcrun at: /Library/Developer/CommandLineTools/usr/bin/xcrun
npm ERR! Darwin 15.2.0
npm ERR! argv "/usr/local/bin/node" "/usr/local/bin/npm" "install" "-g" "airsonos"
npm ERR! node v4.2.3
npm ERR! npm  v2.14.7
npm ERR! code 1

npm ERR! Command failed: git clone --template=/var/root/.npm/_git-remotes/_templates --mirror git://github.com/stephen/node-sonos.git /var/root/.npm/_git-remotes/git-github-com-stephen-node-sonos-git-701294ef
npm ERR! xcrun: error: invalid active developer path (/Library/Developer/CommandLineTools), missing xcrun at: /Library/Developer/CommandLineTools/usr/bin/xcrun
npm ERR! 
npm ERR! 
npm ERR! If you need help, you may report this error at:
npm ERR!     <https://github.com/npm/npm/issues>

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/SAA/npm-debug.log
```

**Comment by aredridel** [comment]
Try fixing your active xcode tools path with `xcode-select`

**Comment by Acili** [comment]
Hi Aria

I have absolutely no idea how to fix the active xcode tools path with xcode-select. What do I do?

**Comment by aredridel** [comment]
try `sudo xcode-select -s /Applications/Xcode.app/Contents/Developer/` if your `Xcode.app` is in `/Applications` and you're on current macos

**Comment by Acili** [comment]
I was obviously missing the xcode so I installed it and re-ran the “npm install -g airsonos” command. A new error popped up about

"xcrun: error: invalid active developer path (/Library/Developer/CommandLineTools), missing xcrun at: /Library/Developer/CommandLineTools/usr/bin/xcrun
so I did the following: xcode-select —install (found from a simple google-search).

Now after running the "npm install -g airsonos” and after 5-10 mins of (scary) output I could do “airsonos” from terminal according to https://medium.com/@stephencwan/hacking-airplay-into-sonos-93a41a1fcfbb#.999di185n <https://medium.com/@stephencwan/hacking-airplay-into-sonos-93a41a1fcfbb#.999di185n>

There is a bit of lag between youtube and the loudspeaker output - but it works!! Thanks for your input.

Cheers
Sadik
